src/components/text-input/index.md.njk

```diff
@@ -102,7 +102,9 @@ Prefixes and suffixes are useful when there's a commonly understood symbol or ab
 
 If you need a specific type of information, say so in the input label or hint text as well. For example, put "Cost, in pounds" in the input label and use the "£" symbol in the prefix.
 
-Some users may miss the fact that the input already has a suffix or prefix, and enter a prefix or suffix into the input. Allow for this in your validation and do not show an error.
+Position prefixes and suffixes so that they're outside of their input. This is to avoid interfering with some browsers that might insert an icon (for example to show or generate a password) or autofill some information into the inputs.
```

Comment:
```suggestion
Position prefixes and suffixes so that they're outside of their input. This is to avoid interfering with some browsers that might insert an icon into the input (for example to show or generate a password).
```